enacts/enactstozarr.py

```diff
@@ -48,53 +37,48 @@ def set_up_dims(xda):
     # and look into xESMF.
     #
     # [1] https://climatedataguide.ucar.edu/climate-data-tools-and-analysis/regridding-overview
-    rr_mrg = rr_mrg.interp(
-        X=np.arange(rr_mrg.X.min(), rr_mrg.X.max() + RESOLUTION, RESOLUTION),
-        Y=np.arange(rr_mrg.Y.min(), rr_mrg.Y.max() + RESOLUTION, RESOLUTION),
-    )
-
-rr_mrg = rr_mrg.chunk(chunks=CHUNKS)
-
-xr.Dataset().merge(rr_mrg).to_zarr(
-  store = RR_MRG_ZARR_PATH
-)
-
-if not os.access(CONFIG["data_dir"], os.W_OK | os.X_OK):
-    sys.exit("can't write to output directory")
-
-def set_up_dims(xda):
-    datestr = Path(xda.encoding["source"]).name.split("_")[2]
-    year = int(datestr[0:4])
-    month = int(datestr[4:6])
-    dekad = int(datestr[6:7])
-    day = (dekad - 1) * 10 + 1
-    xda = xda.expand_dims(T = [dt.datetime(year, month, day)])
-    xda = xda.rename({'Lon': 'X','Lat': 'Y'})
-    return xda
+        print("Your data will be regridded. Refer to function documentation for more information on this.")
+        data = data.interp(
+            X=np.arange(data.X.min(), data.X.max() + resolution, resolution),
+            Y=np.arange(data.Y.min(),data.Y.max() + resolution, resolution),
+        )    
+    return data
 
 def convert(variable):
-    netcdf = list(sorted(Path(CONFIG['data_src'][variable]).glob("*.nc")))
-
+    print(f"converting files for: {variable}")
+    
+    var_name = var_name[2]    
```

Comment:
This doesn't look right. Did you test?

Reply:
@aaron-kaplan No I did not test because of how long it takes for the code to run, and I thought it was trivial. Rémi is going to open a new PR to continue work on this so should he just correct it in his PR? or should I open a quick PR to fix the issue?

Sorry about that..

Reply:
Not urgent, since this isn't being deployed anywhere.

Reply:
Okay, then when @remicousin opens his PR to continue working on this, he can make the change - just for clarification, when you suggested this

`input_path, output_path, var_name = CONFIG['vars'][variable]`

Were you implying making three separate variables directly calling the config var such as ex:

`var_name = CONFIG['vars'][variable][2]`

Reply:
```
input_path, output_path, var_name = CONFIG['vars'][variable]

```
is a more concise way of writing
```
input_path = CONFIG['vars'][variable][0]
output_path = CONFIG['vars'][variable][1]
var_name = CONFIG['vars'][variable][2]
```

Reply:
Oh I see, so then you do not have to put the [0], [1], [2] at the end of each variable. Thank you for the clarification. :)